src/ngraph/runtime/backend.hpp

```diff
@@ -52,6 +52,12 @@ namespace ngraph
                 make_primary_tensor_view(const ngraph::element::Type& element_type,
                                          const Shape& shape) = 0;
 
+            /// @brief Return a handle for a tensor for given mem on backend device
+            virtual std::shared_ptr<ngraph::runtime::TensorView>
+                make_primary_tensor_view(const ngraph::element::Type& element_type,
+                                         const Shape& shape,
+                                         void* mem_handle) = 0;
```

Comment:
use full words. rename to memory_handle, or better yet, memory_pointer as it is a pointer